### ISSUE TITLE
Global rate limiting weight

### ISSUE THREAD

**Issue by TheHolyRoger** [CoinAlpha/hummingbot#3156]
**Feature Suggestion**
It would be nice to have a global rate limit weight/throttling setting per hummingbot instance.

**Impact**
There are times when I run several hummingbot instances at once sharing the same account and the same rate limit - lot's of connectors are now using asyncio Throttler, if there was a global setting to change the throttler settings per instance I could manage my shared rate limit among my running instances.

**Comment by zappra** [comment]
Have you come across rate limit warnings? If so how many instances were you running?

**Comment by TheHolyRoger** [comment]
@zappra I run 8-20 depending, yes I've seen rate limit warnings :)

**Comment by krisj** [comment]
I agree - a global rate limit setting would be fantastic. I recall asking for a "rate meter" function in the HUD display. To show a current call/minute etc. measure. Would give you a good idea of how heavy the bot is firing.

A perpetual strategy I am running can only handle about 2 bots per instance/IP without hitting rate limits on Binance Futs. It is pretty HFT though.  Pure MM and other starts would be running a bit slower I admit. But still I can see 10+ bots per machine/IP needing such a management feature. Buying/configuring extra IPs is no fun :)

**Comment by phbrgnomo** [comment]
Implemented through the Global config `rate_limits_share_pct`. @theholyroger @zappra let us know if this implementation cover your use cases.

Closing for now.
